marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/hub-utils/invoke-queue-write.sjs

```diff
@@ -23,6 +23,9 @@ var contentArray;
 // The collections to add to each content object, based on step definition / flow / step / runtime config
 var configCollections;
 
+// boolean to indicate whether source document collections are preserved in target document
+var targetCollectionsAdditivity;
```

Comment:
Again, this can be part of the context.

Reply:
I don't think it should be part of the context, because then it has to be a part of the context object on every content object being persisted. 

I'm doing some research on this now, as I don't think this module should be in the business of figuring out collections - that should be figured out before we get to this module. It's fine for this module to fiddle with temporal collections, as that's specific to document insertion. 

Also, we want this change to be made in a place so that it's reused by flow.sjs and flowRunner.sjs (and most of flow.sjs is in the process of going into flowRunner.sjs).